### ISSUE TITLE
<Lbl> * can't be identified in adobe accessible checking

### ISSUE THREAD

**Issue by xiuliyouran** [MDPI-AG/luatex-template#64]
@u-fischer  @FrankMittelbach  We use the adobe “Accessibility check” to help check all tags structure, and found we have two points failed. 

**One is about the label * in the front part, see below _screenshot 1._ I suggest we need optimize this point.**

Another is about the table header, for this one, I guess if we want to let this check identify table header, we need give a special table header command for the first row, right?

<img width="1498" height="546" alt="Image" src="https://github.com/user-attachments/assets/ed707ee0-d529-4e25-84f4-4f3132b4e59b" />

<img width="273" height="398" alt="Image" src="https://github.com/user-attachments/assets/57e213fa-4341-46d6-a2aa-b51f78a82a12" />

<img width="523" height="867" alt="Image" src="https://github.com/user-attachments/assets/cdd086e2-b34c-4ac0-a697-8d990125a53b" />

**Comment by FrankMittelbach** [comment]
> [@u-fischer](https://github.com/u-fischer) [@FrankMittelbach](https://github.com/FrankMittelbach) We use the adobe “Accessibility check” to help check all tags structure, and found we have two points failed.

Pains me to say, but you can't use Adobe's accessibility checker with PDF/UA-2 files. It simply reports bogus errors as it only understands PDF/UA-1 and checks against the UA-1 rules. Same problem with some other checkers like PAC.

See https://github.com/latex3/tagging-project/discussions/categories/issues-with-accessibility-checkers-and-other-at-software?discussions_q=is%3Aopen+category%3A%22Issues+with+accessibility+checkers+and+other+AT+software%22+label%3A%22spurious+validation+error+%5BAdobe%5D%22 for a collections of issues with Adobe's checker.

You need to use a UA-2 enabled checker in your workflow, e.g., VeraPDF!

What also would help is if you as a customer of Adobe, complain about their software not supporting the UA-2 standard.
 
> **One is about the label * in the front part, see below _screenshot 1._ I suggest we need optimize this point.**

It thinks that Lbl is only allowed as part of a List, which is not the case.  In UA-2 it can be used in various elements because it means "label" not "list-label".


> 
> Another is about the table header, for this one, I guess if we want to let this check identify table header, we need give a special table header command for the first row, right?

I can't check right now (and your screen shots do not show) what it's complaint about  the table header is, but this is probably also bogus. It probably misinterprets a presentation table somewhere.

**Comment by xiuliyouran** [comment]
@FrankMittelbach Thank you for your further explanation about this issue. Yes, I has switched to use https://demo.verapdf.org/ to validate. And just make a double check with adobe accessibility function. If we're sure that we can only follow the result that from verapdf, ignore the result from adobe, then we have no query about this. Thank you for your help.

**Comment by FrankMittelbach** [comment]
As I said, it would be very helpful if you as a bigger publishing house complain at Adobe that their software is not correctly handling PDF/UA-2 and that you want to be able to use their software.

**Comment by u-fischer** [comment]
As said by Frank, this is a problem of the adobe checker, so I'm closing.

**Comment by FrankMittelbach** [comment]
@xiuliyouran @rordi Since you are a customer of Adobe's software it I suggest you make a bug report with them (if you want to use that functionality) and ask them when they support PDF/UA-2 properly and not generate bogus warnings in their checker, because that makes their software unusable for you.
